db/migrate/20230607063136_create_comments.rb

```diff
@@ -0,0 +1,15 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :user_id
```

Comment:
- [ ] @adamilare great work adding the comments table, however, the reference to the user should be authorid as it is in the ERD diagram,
![image](https://github.com/adamilare/railsblog/assets/61408860/db77163e-ac61-4900-a119-aa457639e812)

Reply:
In addition to my reply to your remark under 'like' migration file, here is the specific statement differentiating the 'AuthorId' in Post from the rest;

`Pay attention that in Post table the column holding the foreign key to the users table should be named author_id`

https://github.com/microverseinc/curriculum-rails/blob/41a60e86aabd76089746aeae8b348dd61a091908/blog-app/projects/creating_data_model_project_v1_2.md?plain=1#L36